recipes/icon/build.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/env bash
+
+# Abort script upon untested failure
+set -e
+
+make Configure name=linux
+
+make CC=${CC}
+
+# Run a small sample of the test suite
+make Samples
+
+# Deploy to icon and bin directories at PREFIX
+mkdir -p ${PREFIX}/bin
+DEPLOY=${PREFIX}/icon
+mkdir -p ${DEPLOY}
+
+# remove compilation intermediates
+make Clean || echo Clean exit code is $?
+
+# Install by copy to give users full access to source, e.g., so that they can
+#   use it for reference when building loadable C functions.
+cp -R ${SRC_DIR}/* ${DEPLOY}
+# Clean out extra files if necessary
+pushd ${DEPLOY}
+if [ -f build_env_setup.sh ]; then
+  rm build_env_setup.sh
+fi
+if [ -f conda_build.sh ]; then
+  rm conda_build.sh
+fi
+if [ -f metadata_conda_debug.yaml ]; then
+  rm metadata_conda_debug.yaml
+fi
+popd
+
+# Put executables onto the path
+(pushd ${PREFIX}/bin && ln -s ../icon/bin/* .)
+
+# Create or append activation script to set IPL envar and give tips
+mkdir -p ${PREFIX}/etc/conda/activate.d
+echo '#!/bin/sh
+if [ -d ${CONDA_PREFIX}/icon/ipl ]; then
+  export IPL_OLD=${IPL}
+  export IPL=${CONDA_PREFIX}/icon/ipl
+  cat << .
+    For info regarding the Icon programming language, please see
+      https://www.cs.arizona.edu/icon
+
+    The Icon Programing Library is at \${IPL}:
+      ${IPL}
+
+    For offline help for icon:
+      man -l \$IPL/../man/man1/icon.1
+
+    For offline help for icont and iconx:
+      man -l \$IPL/../man/man1/icont.1
+
+    To build loadable C functions, see:
+      `readlink -f ${IPL}/../doc/condagcc.txt`
+.
```

Comment:
activate scripts are not for informational messages. it becomes annoying to users.

Reply:
I will accept your statement even though I find it bewildering when I activate and get no hints.

Perhaps I can point the man path to the man pages at least ...